mypy/server/update.py

```diff
@@ -852,9 +852,6 @@ def key(node: DeferredNode) -> int:
 
     nodes = sorted(nodeset, key=key)
 
-    # TODO: ignore_all argument to set_file_ignored_lines
-    manager.errors.set_file_ignored_lines(file_node.path, file_node.ignored_lines)
```

Comment:
It looks like this change breaks the `# type: ignore` in fine grained mode.

Reply:
Argh I thought I had run the tests >_>